gubernator.go

```diff
@@ -296,6 +295,10 @@ func (s *Instance) getRateLimit(r *RateLimitReq) (*RateLimitResp, error) {
 		s.global.QueueUpdate(r)
 	}
 
+	if HasBehavior(r.Behavior, Behavior_MULTI_REGION) {
+		s.conf.RegionPicker.QueueHits(r)
```

Comment:
The RegionPicker should not `QueueHits`, we should make a `multiRegionManager` to handle this. The `multiRegionManager` will use the `RegionPicker` to decide what nodes in other regions to send hits too.